doc/getting-started/getting-started.rst

```diff
@@ -825,7 +825,7 @@ Launch the User VM
    .. code-block:: console
 
       ubuntu@ubuntu:~$ uname -r
-      5.15.0-43-generic
+      5.15.71-acrn-service-vm-00001-g7df95512ae9e
```

Comment:
When following the GSG, I see the uname output for the service VM as simply "5.15.71-acrn-service-vm" not what you're showing.